src/store/modules/segmentationModule.js

```diff
@@ -0,0 +1,1411 @@
+import external from '../../externalModules.js';
+import { getToolState } from '../../stateManagement/toolState.js';
+import getNewColorLUT from '../../util/segmentation/brush/getNewColorLUT.js';
+import labelmapStats from '../../util/segmentation/brush/labelmapStats.js';
+import EVENTS from '../../events.js';
+
+import { getters as storeGetters } from '../index.js';
+
+import { getLogger } from '../../util/logger.js';
+import pointInImage from '../../util/pointInImage.js';
+
+const logger = getLogger('store:modules:segmentationModule');
+
+// Internal state of the brush module.
+const state = {
+  colorMapId: 'BrushColorMap',
+  series: {},
+  colorLutTables: {},
+};
+
+const configuration = {
+  renderOutline: true,
+  renderFill: true,
+  renderInactiveLabelmaps: true,
+  radius: 10,
+  minRadius: 1,
+  maxRadius: 50,
+  segmentsPerLabelmap: 65535, // Max is 65535 due to using 16-bit Unsigned ints.
+  fillAlpha: 0.2,
```

Comment:
Can this be overridden per segmentation per labelmap?

Reply:
Nope, but would you really want to have different settings per labelmap? 🤔 . I don't think slicer does this, but maybe I'm wrong. They feel like they should be global settings, but I'm happy to discuss if you can give a good example of usage you've seen which indicates otherwise.

You can turn off segments individually for a particular labelmap, though.